### DISCUSSION TITLE
How disable auto login after registration ?

### DISCUSSION THREAD

**Discussion by f2yez** [laravel/framework#34583] [General]
Laravel Version: 8

How I can disable auto login after registration ?

![](https://user-images.githubusercontent.com/7463995/94578630-a4ddc980-0280-11eb-9d85-c6c18f86573c.png)

I think I needs to override store method, but how ?

**Comment by snapey** [comment]
Why would you want to do this?  Not because you are an admin creating other users I hope?

two options I can think of;
1. Create your own registration controller and link your form to its store input.  Perform the same lines as in the Fortify code apart from the login.

2. Leave the fortify code, but create and register your own RegisterResponse class that logs the user out again.

If you _are_ creating users as part of an admin function then just CRUD directly on the users table. Do not consider this a case for 'registration'

**Reply by diegoldev** [reply]
Hi @snapey 
> Why would you want to do this? Not because you are an admin creating other users I hope?

What about if you need the user verify his/her email before can access to protected routes?

**Reply by mateusjunges** [reply]
@diegoldev If a user needs to verify his or her email before he can access protected routes, you can just follow [this documentation](https://laravel.com/docs/8.x/verification), then just protect your routes where the user must have a verified email with the `verified` middleware and you are good to go, there is no need to logout the user after the registration.

**Reply by diegoldev** [reply]
Thanks @mateusjunges !! Didn't know about that.

**Comment by Maycomsantos** [comment]
Other form is desactive in routes/web.php 

 you can use something like this: 
![carbon (1)](https://user-images.githubusercontent.com/33041694/114939246-8d2df300-9e16-11eb-8006-607d33b3cbb7.png)

**Reply by mateusjunges** [reply]
@Maycomsantos It will not disable auto login after the registration. 
Also, your syntax is wrong here. The correct code is this:

```php
Auth::routes(['register' => false]);
```

It only disable the register route, not the auto login after registration.

**Reply by Maycomsantos** [reply]
Thank's you... well noted
